resources/definitions/fdmprinter.def.json

```
@@ -1120,10 +1116,8 @@
                             "type": "float",
                             "unit": "%",
                             "default_value": 50,
-                            "value": "wall_transition_threshold",
+                            "value": "wall_transition_threshold * 8/9",
```

```suggestion
                            "value": "wall_transition_threshold * 8 / 9",
```
Nitpicking, operators should have white spaces around them.

---

Will change